Bluepill-runner/Bluepill-runner/BPRunner.m

```diff
@@ -274,7 +274,14 @@ - (int)runWithBPXCTestFiles:(NSArray<BPXCTestFile *> *)xcTestFiles {
             [BPUtils printInfo:INFO withString:@"%lu Simulator%s still running. [%@]",
              launchedTasks, launchedTasks == 1 ? "" : "s", listString];
             [BPUtils printInfo:INFO withString:@"Using %d of %d processes.", numprocs(), maxProcs];
-
+            if (numprocs() > maxProcs * BP_MAX_PROCESSES_PERCENT) {
```

Comment:
pretty sure maxProcs * 0.75 = 0

Reply:
Well, it does not seem so. maxProcs is an integer, and the result is assigned to another integer. It will only round to an integer.

Reply:
You're right!